University-advisor/Forms/MainForm.cs

```diff
@@ -295,13 +148,32 @@ private void UniversitiesGrid_CellClick(object sender, DataGridViewCellEventArgs
         private void ReviewSubmit_Click(object sender, EventArgs e)
         {
             string review = universityReview.Text;
-            if(review.Length == 0)
+            int rating = 0;
+            if (veryBadButton.Checked)
```

Comment:
It is not great when you mix if/else statement structure. Most of the code was written using structure as in 233-238 lines. So lets stick to it.

Reply:
Fixed.